### ISSUE TITLE
Pathway False Positive

### ISSUE THREAD

**Issue by Immedicable** [xaw3ep/NoCheatPlus#41]
18.03 12:44:19 [Server] INFO ---- Version information ----
18.03 12:44:19 [Server] INFO #### Server ####
18.03 12:44:19 [Server] INFO git-Paper-137 ~MC: 1.15.2~
18.03 12:44:19 [Server] INFO detected: 1.15.2
18.03 12:44:19 [Server] INFO #### NoCheatPlus ####
18.03 12:44:19 [Server] INFO Plugin: 3.17-SNAPSHOT-b5.6.2
18.03 12:44:19 [Server] INFO MCAccess: 1.13-1.15.2|? / Bukkit-API
18.03 12:44:19 [Server] INFO Features:
18.03 12:44:19 [Server] INFO blocks: BlocksMC1_4 | BlocksMC1_5 | BlocksMC1_6_1 | BlocksMC1_7_2 | BlocksMC1_8 | BlocksMC1_9 | BlocksMC1_10 | BlocksMC1_11 | BlocksMC1_12 | BlocksMC1_13 | BlocksMC1_14 | BlocksMC1_15 | MCAccessBukkitModern
18.03 12:44:19 [Server] INFO checks: FastConsume | Gutenberg | HotFixFallingBlockPortalEnter | AttackFrequency | FlyingFrequency | KeepAliveFrequency
18.03 12:44:19 [Server] INFO defaults: pvpKnockBackVelocity
18.03 12:44:19 [Server] INFO packet-listeners: UseEntityAdapter | MovingFlying | OutgoingPosition | KeepAliveAdapter | SoundDistance | NoSlow
18.03 12:44:19 [Server] INFO Hooks: mcMMO~cncp~ 2.3 | ViolationFrequency~NCP~ 1.0
18.03 12:44:19 [Server] INFO #### Related Plugins ####
18.03 12:44:19 [Server] INFO CompatNoCheatPlus v6.6.5-RC-sMD5NET-b90 | ProtocolLib v4.5.0-SNAPSHOT-b437 | ConditionalCommands v1.5 | ConsoleSpamFix v1.8.1

When walking on a pathway, you get glitched as the anticheat mistakes it for phase.

**Comment by MarkElf** [comment]
Please specify what you mean by a pathwayand provide a debug log (/ncp debug player playername).

**Comment by Immedicable** [comment]
Yeah, so I mean the pathway block (I think that's what it's called). You make it by right-clicking grass with a shovel.

Also, I'm not sure if this is the debug data, but here goes:

20-03-18 16:23:25 [FINE] [MOVING_NOFALL] [TrialFire] NoFall: mc=0.0 / nf=0.0 | ymax=70.0
20-03-18 16:23:25 [FINE] [NET] [TrialFire] Flying(ground=true,pitch=28.471539,yaw=160.4517)
20-03-18 16:23:25 [FINE] [NET] [TrialFire] Flying(ground=true,x=4552.885386324236,y=70.0,z=4757.543121960119)
20-03-18 16:23:26 [FINE] [NET] [TrialFire] Flying(ground=true,x=4552.885386324236,y=70.0,z=4757.553890463111)
20-03-18 16:23:27 [FINE] [NET] [TrialFire] Flying(ground=true,x=4552.885386324236,y=70.0,z=4757.553890463111)
20-03-18 16:23:28 [FINE] [NET] [TrialFire] Flying(ground=true,x=4552.885386324236,y=70.0,z=4757.553890463111)
20-03-18 16:23:29 [FINE] [NET] [TrialFire] Flying(ground=true,x=4552.885386324236,y=70.0,z=4757.553890463111)
20-03-18 16:23:30 [FINE] [NET] [TrialFire] Flying(ground=true,x=4552.885386324236,y=70.0,z=4757.553890463111)
20-03-18 16:23:31 [FINE] [NET] [TrialFire] Flying(ground=true,x=4552.885386324236,y=70.0,z=4757.553890463111)
20-03-18 16:23:32 [FINE] [NET] [TrialFire] Flying(ground=true,x=4552.885386324236,y=70.0,z=4757.553890463111)
20-03-18 16:23:33 [FINE] [NET] [TrialFire] Flying(ground=true,x=4552.885386324236,y=70.0,z=4757.553890463111)
20-03-18 16:23:34 [FINE] [NET] [TrialFire] Flying(ground=true,x=4552.885386324236,y=70.0,z=4757.553890463111)
20-03-18 16:23:35 [FINE] [NET] [TrialFire] Flying(ground=true,x=4552.885386324236,y=70.0,z=4757.553890463111)
20-03-18 16:23:36 [FINE] [NET] [TrialFire] Flying(ground=true,x=4552.885386324236,y=70.0,z=4757.553890463111)
20-03-18 16:23:37 [FINE] [NET] [TrialFire] Flying(ground=true,x=4552.885386324236,y=70.0,z=4757.553890463111)
20-03-18 16:23:38 [FINE] [NET] [TrialFire] Flying(ground=true,x=4552.885386324236,y=70.0,z=4757.553890463111)

**Comment by MarkElf** [comment]
Oh, path blocks. Turns out I forgot to add this commit to the compiled build. Send me a message on MC-Market so I can recompile it and send it to you.
https://github.com/MarkElf/NoCheatPlus/commit/8aa9ef8ab568d09f02d13488c243c220c5b4891e
